styles/nav.scss

```diff
@@ -119,7 +119,7 @@ $daptiv-subnav-tab-h-pad: 14px;
         }
     }
 
-    .daptiv-nav-menuitem-alternate {
+    .daptiv-nav-menuitem.alternate {
```

Comment:
The only construct between the 'alternate' rule and daptiv-nav-menuitem is the 'disabled' rule on the same class. Do you have some guidelines on ordering of these 'decorator' rules? 
Have removed the @extend rule.